terra-jupyter-base/Dockerfile

```diff
@@ -140,6 +138,7 @@ RUN chown -R $USER:users $JUPYTER_HOME \
 
 USER $USER
 EXPOSE $JUPYTER_PORT
+WORKDIR $HOME
```

Comment:
The new base image's WORKDIR is `/` for some reason.. setting it to `$HOME` which is what it was before.